<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandWindow</class>
 <widget class="QMainWindow" name="CommandWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>491</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="DeleteCommand">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MoveDown">
         <property name="text">
          <string>Move Down</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MoveUp">
         <property name="text">
          <string>Move Up</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="CommandsRun">
         <property name="text">
          <string>x/x</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>118</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="TimeEllapsed">
         <property name="text">
          <string>0:00 elapsed</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>305</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulator">
    <property name="title">
     <string>Simulator</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionClear"/>
    <addaction name="actionPause"/>
    <addaction name="actionForward"/>
    <addaction name="actionBackward"/>
   </widget>
   <addaction name="menuSimulator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionClear"/>
   <addaction name="actionBackward"/>
   <addaction name="actionPause"/>
   <addaction name="actionForward"/>
  </widget>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/play.png</normaloff>:/Icon Storage/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Runs the simulation</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/reset.png</normaloff>:/Icon Storage/reset.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>clear simulation</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/next.png</normaloff>:/Icon Storage/next.png</iconset>
   </property>
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="toolTip">
    <string>steps forward</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/previous.png</normaloff>:/Icon Storage/previous.png</iconset>
   </property>
   <property name="text">
    <string>Backward</string>
   </property>
   <property name="toolTip">
    <string>step backward</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/Icon Storage/pause.png</normaloff>:/Icon Storage/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>pause simulation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
</ui>
